lms/djangoapps/ccx/views.py

```diff
@@ -238,6 +241,14 @@ def create_ccx(request, course, ccx=None):
     for rec, response in responses:
         log.info('Signal fired when course is published. Receiver: %s. Response: %s', rec, response)
 
+    # .. event_implemented_name: COURSE_CREATED
+    COURSE_CREATED.send_event(
+        time=datetime.datetime.now(tz=timezone.utc),
+        course=CourseData(
+            course_key=ccx,
```

Comment:
```suggestion
            course_key=ccx_id,
```
This is not necessary https://github.com/Pearson-Advance/edx-platform/pull/176/commits/b22987703f90b515d0ae18ed151545590e6934f6

Please verify the last change in the olive branch